grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java

```diff
@@ -160,18 +158,20 @@ private void loadAndUnlock(Lock lock) {
 
     /**
      * Check in with the properties file to decide which type of task manager should be started
-     * @param inMemoryQueue
-     * @param redisCountStorage
+     * and return the TaskManager
+     * @param inMemoryQueue         True if running in memory
      * @param jedisPool
      */
     private TaskManager startTaskManager(
             boolean inMemoryQueue,
-            RedisCountStorage redisCountStorage,
-            Pool<Jedis> jedisPool, LockProvider lockProvider) {
+            Pool<Jedis> jedisPool,
+            LockProvider lockProvider) {
         TaskManager taskManager;
         if (!inMemoryQueue) {
             taskManager = new RedisTaskManager(engineId, prop, jedisPool, factory, lockProvider, metricRegistry);
         } else  {
+            // Redis storage for counts, in the RedisTaskManager it's created in consumers
+            RedisCountStorage redisCountStorage = RedisCountStorage.create(jedisPool);
```

Comment:
Aren't the count's always stored in redis? Why the need for this wrapper only for `StandaloneTaskManager `?